azure-pipelines.yml

```diff
@@ -0,0 +1,88 @@
+name: $(BuildDefinitionName)_$(Date:yyyyMMdd)$(Rev:.rr)
+variables:
+  CI_NAME: Azure Pipelines
+  CI_BUILD_ID: $(Build.BuildId)
+  CI_BUILD_URL: "https://dev.azure.com/plasmapy/plasmapy/_build/results?buildId=$(Build.BuildId)"
+  CIBW_BUILD: cp37-* cp38-*
+  CIBW_SKIP: "*-win32 *-manylinux1_i686"
+
+resources:
+  repositories:
+  - repository: OpenAstronomy
+    type: github
+    endpoint: PlasmaPy-NEI
```

Comment:
Turns out this should have been just `PlasmaPy`. It probably references the org, not the repo. Or something. ^^`